src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -1111,22 +1086,23 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
             if (ObsBias):
                 valuebc = [
                     "constant_{:d}".format(a),
-                    "scan_angle_{:d}".format(a),
+                    "zenith_angle_{:d}".format(a),
                     "cloud_liquid_water_{:d}".format(a),
                     "lapse_rate_squared_{:d}".format(a),
                     "lapse_rate_{:d}".format(a),
                     "cosine_of_latitude_times_orbit_node_{:d}".format(a),
                     "sine_of_latitude_{:d}".format(a),
                     "emissivity_{:d}".format(a),
-                    "scan_angle_4th_order_{:d}".format(a),
-                    "scan_angle_3rd_order_{:d}".format(a),
-                    "scan_angle_2nd_order_{:d}".format(a),
-                    "scan_angle_1st_order_{:d}".format(a),
+                    "scan_angle_order_4_{:d}".format(a),
+                    "scan_angle_order_3_{:d}".format(a),
+                    "scan_angle_order_2_{:d}".format(a),
+                    "scan_angle_{:d}".format(a),
                 ]
                 ibc = 0
                 for vbc in valuebc:
                     varDict[vbc]['bctKey'] = vbc, writer.ObiastermName()
                     varDict[vbc]['bcpKey'] = vbc, writer.ObiaspredName()
+                    print("emily checking valuebc 1 = ", valuebc)
```

Comment:
Remove this comment

Reply:
@CoryMartin-NOAA  Oop!  Thanks for catching this. Fix is on the way...